packages/amplify-cli/src/extensions/amplify-helpers/update-backend-config.js

```diff
@@ -4,6 +4,7 @@ const pathManager = require('./path-manager');
 function updateBackendConfigAfterResourceAdd(category, resourceName, options) {
   const backendConfigFilePath = pathManager.getBackendConfigFilePath();
   const backendConfig = JSON.parse(fs.readFileSync(backendConfigFilePath));
+  if (options.output) delete options.output;
```

Comment:
Why do we need this?

Reply:
This can be removed now because we wired up cloud formation.